docs/designs/xref.md

```diff
@@ -91,8 +91,42 @@ xref:
   - http://url1/.xrefmap.json
   - http://url2/.xrefmap.json
 ```
+User should be able to set a string instead of array, docfx will expand it to an array during the build.\
 During `docfx restore`, all the JSON files will be restored and merged.
 
+#### Xref base path
+Since we do not have a clear picture how we are going to support the local build, we will focus on OPS build firstly.\
+Unlike DocFX v2, the user of v3 needs to set where to get the `.xrefmap.json` files in `docfx.yml`.
```

Comment:
Should this be in `.openpublish.config.json`? `docfx.yml` currently does not exist in repo.

Reply:
Actually, it is in `.openpublishing.publish.config.json` already, maybe there is no need to do the migration